docs/configuration.md

```diff
@@ -1158,6 +1158,10 @@ Apart from these, the following properties are also available, and may be useful
     For more detail, see the description
     <a href="job-scheduling.html#dynamic-resource-allocation">here</a>.
     <br><br>
+    When this property is enabled, <code>spark.executor.instances</code> will affect the initial
+    number of executors, increasing the initial number if it is higher than the minimum or the
```

Comment:
I think the wording here is a bit confusing.    Perhaps remove everything after the , and just say see spark.dynamicAllocation.initialExecutors and we can add more description there.